app/uk/gov/hmrc/soletraderidentificationfrontend/views/helpers/inputText.scala.html

```diff
@@ -23,6 +23,7 @@
     isPageHeading: Boolean,
     hasFieldTitle: Boolean = false,
     visibleLabel: Boolean = false,
+    mediumLabel: Boolean = false,
```

Comment:
I made a recent change to do this for the label. Does this cover what you need? Might need a rebase.

https://github.com/hmrc/sole-trader-identification-frontend/blob/main/app/uk/gov/hmrc/soletraderidentificationfrontend/views/helpers/inputText.scala.html#L41